### DISCUSSION TITLE
How to initialize a client with a relative address?

### DISCUSSION THREAD

**Discussion by cawa-93** [trpc/trpc#2288] [Q&A]
We have an application that runs on multiple domains. To avoid cross-origin requests, the client needs to be initialized not with an absolute, but with a relative address
```ts
const client = createTRPCClient<AppRouter>({
  url: '/trpc', // <-- Like this
});
```

**Comment by romanzy313** [comment]
You can grab `window.location.host` and append /trpc to it

Something like this should do the trick 
```javascript
const client = createTRPCClient<AppRouter>({
  url: `${window.location.host}/trpc`
});
```

**Reply by cawa-93** [reply]
What about ssr?

**Reply by romanzy313** [reply]
This one i'm not familiar with, the answer really depends on the framework. Which one are you using?

**Reply by cawa-93** [reply]
https://github.com/wobsoriano/trpc-nuxt
